### PR TITLE
Control: compute the acceleration, jerk variables for Mpc Controller

### PR DESCRIPTION
**Pull request by kevin-y-wang**
Similar modifications on MPC controller, as on lon/lat controllers. 
Generated records already verified by Dreamland simulation

### CODE REVIEW

File: modules/control/controller/mpc_controller.cc [hunk 1]
```diff
@@ -575,21 +576,61 @@ void MPCController::ComputeLateralErrors(
...
+      lateral_error_dot = -lateral_error_dot;
+      lateral_error_dot_dot = -lateral_error_dot_dot;
+    }
+  }
+
+  debug->set_lateral_error_rate(lateral_error_dot);
+
+  debug->set_lateral_acceleration(lateral_error_dot_dot);
+
+  debug->set_lateral_jerk(
```
Comment (xiaoxq) [inline modules/control/controller/mpc_controller.cc]: Remove the 2 blank rows above.

File: modules/control/controller/mpc_controller.cc [hunk 2]
```diff
@@ -575,21 +576,61 @@ void MPCController::ComputeLateralErrors(
...
 
   // theta_error = delta_theta;
   debug->set_heading_error(delta_theta);
   // theta_error_dot = angular_v - matched_point.path_point().kappa() *
   // matched_point.v();
-  debug->set_heading_error_rate(angular_v - matched_point.path_point().kappa() *
-                                                matched_point.v());
+  debug->set_heading_rate(angular_v);
+  debug->set_ref_heading_rate(matched_point.path_point().kappa() *
+                              matched_point.v());
```
Comment (xiaoxq) [inline modules/control/controller/mpc_controller.cc]: Better to put statement in one line, such as `matched_point.path_point().kappa() * matched_point.v()`

File: modules/control/controller/mpc_controller.cc [hunk 3]
```diff
@@ -625,16 +666,39 @@ void MPCController::ComputeLongitudinalErrors(
 
   ADEBUG << "matched point:" << matched_point.DebugString();
   ADEBUG << "reference point:" << reference_point.DebugString();
-  debug->set_station_error(reference_point.path_point().s() - s_matched);
-  debug->set_speed_error(reference_point.v() - s_dot_matched);
+
+  double heading_error = common::math::NormalizeAngle(
+      VehicleStateProvider::Instance()->heading() - matched_point.theta());
+  double lon_speed = VehicleStateProvider::Instance()->linear_velocity() *
```
Comment (xiaoxq) [inline modules/control/controller/mpc_controller.cc]: Make pointer or reference for long names, especially if they are used more than once.
`auto* linear_velocity = VehicleStateProvider::Instance()->linear_velocity();`
Shorter lines are more readable.

**Comment by kevin-y-wang** [comment]
Modifications finished per every comment

**Pull request merged**